static/README.md

```diff
@@ -0,0 +1,94 @@
+# Raster Foundry
+
+A complete, yet simple, starter for Angular using Webpack.
```

Comment:
These notes are kind of generic and look like they might be leftover. We can probably leave them for now as I'll be replacing it later with my PR on getting a development environment set up.